src/main/scala/fi/oph/koski/schema/DIA.scala

```diff
@@ -103,7 +103,7 @@ trait DIASuoritus extends Vahvistukseton
 case class DIAOppiaineenValmistavanVaiheenSuoritus(
   @Description("DIA-oppiaineen tunnistetiedot")
   @Title("Oppiaine")
-  koulutusmoduuli: DIAOsaAlueOppiaine,
+  koulutusmoduuli: DIATutkintoOppiaine,
```

Comment:
Tässä taitaa olla ajatus tai nimeäminen väärinpäin: nyt käytetään _tutkintoainetta_ _valmistavassa suorituksessa

Reply:
Totta. Nyt nimetty järkevämmin: `DIATutkintoOppiaine` --> `DIAValmistavanVaiheenOppiaine`.